pkg/bbgo/persistence.go

```diff
@@ -27,6 +27,10 @@ func (p *Persistence) backendService(t string) (service.PersistenceService, erro
 		return p.Facade.Json, nil
 
 	case "redis":
+		// fallback to memory persistence
+		if p.Facade.Redis == nil {
```

Comment:
Can we add a warning log here? "redis persistence is not available, fallback to memory backend"